docs/book/kubernetes-admission-control.md

```diff
@@ -27,7 +27,7 @@ Start minikube with these admission controllers enabled:
 
 ```bash
 minikube start --kubernetes-version v1.9.0 \
-  --extra-config=apiserver.Admission.PluginNames=$ADMISSION_CONTROLLERS
+  --extra-config=apiserver.admission-control=$ADMISSION_CONTROLLERS
```

Comment:
It looks like in v1.10, the validating admission controller is enabled by default. What I'd rather do here is change this line to `minikube start` and update the prerequisites section to say that the tutorial is supported with minikube version `v0.28.0` or greater.

Reply:
I just check actually in v1.9 the admission controller was also enabled by default (at least when you bootstrap a cluster with minikube).

I changed the docs like you described.